packages-ts/gauntlet-solana-contracts/src/commands/contracts/ocr2/setConfig.ts

```diff
@@ -4,44 +4,61 @@ import { SolanaCommand, TransactionResponse } from '@chainlink/gauntlet-solana'
 import { PublicKey } from '@solana/web3.js'
 import BN from 'bn.js'
 import { CONTRACT_LIST, getContract } from '../../../lib/contracts'
+import { getRDD } from '../../../lib/rdd'
 
 export default class SetConfig extends SolanaCommand {
   static id = 'ocr2:set_config'
-  static category = CONTRACT_LIST.ACCESS_CONTROLLER
+  static category = CONTRACT_LIST.OCR_2
 
   static examples = [
     'yarn gauntlet ocr2:set_config --network=devnet --state=EPRYwrb1Dwi8VT5SutS4vYNdF8HqvE7QwvqeCCwHdVLC',
-    'yarn gauntlet ocr2:set_config --network=devnet --state=5oMNhuuRmxPGEk8ymvzJRAJFJGs7jaHsaxQ3Q2m6PVTR EPRYwrb1Dwi8VT5SutS4vYNdF8HqvE7QwvqeCCwHdVLC',
   ]
+
+  makeInput = (userInput): Input => {
+    if (userInput) return userInput as Input
```

Comment:
Rebased onto my testing branch, and just testing things out. Looks like this fails with 
```$ yarn build && pkg .
$ yarn clean && tsc -b ./tsconfig.json
$ tsc -b --clean ./tsconfig.json
packages-ts/gauntlet-solana-contracts/src/commands/contracts/ocr2/setConfig.ts:17:28 - error TS2304: Cannot find name 'Input'.

17   makeInput = (userInput): Input => {
                              ~~~~~

packages-ts/gauntlet-solana-contracts/src/commands/contracts/ocr2/setConfig.ts:18:40 - error TS2304: Cannot find name 'Input'.

18     if (userInput) return userInput as Input
                                          ~~~~~


Found 2 errors.
```

Reply:
Thanks! Last rebase removed it